src/esportsbot/reactionMenus/reactionPollMenu.py

```diff
@@ -0,0 +1,265 @@
+from __future__ import annotations
+from . import reactionMenu
+from .. import lib
+from discord import Colour, Message, Embed, User, Member
+from typing import Dict, Union
+
+
+BALLOT_BOX_IMAGE = "https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/120/twitter/259/ballot-box-with-ballot_1f5f3.png"
+
+
+def makePollBar(name: str, numVotes: int, maxNameLength: int, maxVotes: int, maxBarLength: int) -> str:
+    """Make a bar for a poll results bar chart, for the statistics of a given poll option.
+
+    :param str name: The name of the poll option
+    :param int numVotes: The number of votes that the option received
+    :param int maxNameLenth: The length of the longest option name in the poll
```

Comment:
makePollBar no longer has a paramete called maxNameLenth, can be removed from docstring.